### PR TITLE
CSHARP-4203: IFindFluentExtensions for Any and AnyAsync

### PR DESCRIPTION
**Pull request by mhdejonge**
[Jira Link](https://jira.mongodb.org/browse/CSHARP-4203)

### CODE REVIEW

File: tests/MongoDB.Driver.Tests/IFindFluentExtensionsTests.cs [hunk 1]
```diff
@@ -28,6 +29,65 @@ namespace MongoDB.Driver.Tests
...
+            mockCursor.SetupGet(c => c.Current).Returns(firstBatch);
+
+            bool result;
+            if (async)
+            {
+                mockSubject3.Setup(s => s.ToCursorAsync(cancellationToken)).Returns(Task.FromResult(mockCursor.Object));
+                mockCursor.Setup(c => c.MoveNextAsync(cancellationToken)).Returns(Task.FromResult(true));
+
+                result = mockSubject1.Object.AnyAsync(cancellationToken).GetAwaiter().GetResult();
+
```
Comment (BorisDog) [inline tests/MongoDB.Driver.Tests/IFindFluentExtensionsTests.cs:58]: Unnecessary empty line

**Review by BorisDog — CHANGES_REQUESTED** [review]
Looks great, left a minor comment.

File: tests/MongoDB.Driver.Tests/IFindFluentExtensionsTests.cs (hunk 1 above)
Comment (mhdejonge-tby) [reply tests/MongoDB.Driver.Tests/IFindFluentExtensionsTests.cs]: Removed it.

Comment (mhdejonge) [reply tests/MongoDB.Driver.Tests/IFindFluentExtensionsTests.cs]: Removed it.

**Review by rstam — APPROVED** [review]
LGTM

**Pull request merged**